gcn_classic_to_json/notices/GECAM_GND/__init__.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from scipy.stats import norm
+
+from ... import utils
+
+src_class_vals = ["GRB", "SFLARE", "KNOWN_SOURCE", "GENERIC_SOURCE"]
+
+containment_prob = norm().cdf(1) - norm.cdf(-1)
+
+
+def parse(bin):
+    bin[15]  # Unused. According to docs: '4 bytes for the future'
+    bin[20:39]  # Unused. According to docs: '76 bytes for the future'
+
+    soln_status_bits = np.flip(np.unpackbits(bin[18:19].view(dtype="u1")))
+    comments = None
+    if soln_status_bits[0] == 1:
+        comments = "This is a test notice."
```

Comment:
We should capture this in a machine-readable way, not as a comment.

Reply:
I can add 'alert_tense' for this, should I remove the comment?

Reply:
Yes.